client/scss/components/_listing.scss

```diff
@@ -487,6 +487,60 @@ ul.listing {
     }
 }
 
+footer.bulk-actions-choices {
+    @include transition(bottom 0.5s ease 0.5s);
```

Comment:
This is very slow. Maybe just `bottom 0.1s`

Reply:
The action bar requires further work on mobile. Please use the edit form action bar as a guide.

![Screenshot 2021-07-14 at 16 54 26](https://user-images.githubusercontent.com/31622/125653179-58670dd4-4d9b-4730-97e8-51baef0c8382.png)
![Screenshot 2021-07-14 at 16 54 18](https://user-images.githubusercontent.com/31622/125653467-ad0b6d22-fa2c-43e5-acf0-21490a5b4e0d.png)

The delete button hover text is all white, so unreadable :/ There's an existing issue with the wrong colours on hover in listings (#7143, fixed in #7302) so worth checking the solution and the overlap there